nservicebus/hosting/nservicebus-host/index.md

```diff
@@ -33,17 +33,17 @@ Versions of the host prior to 5.0 were aligned with NServiceBus core. Since vers
 
 ## Application Domains
 
-The `NServiceBus.Host.exe` creates a separate *service* [Application Domain](https://docs.microsoft.com/en-us/dotnet/framework/app-domains/application-domains) to run NServiceBus and the user code. The new domain is assigned a configuration file named after the dll that contains the class implementing `IConfigureThisEndpoint`. All the configuration should be done in that file (as opposed to `NServiceBus.Host.exe.config`). In most cases that means just adding the `app.config` file to the project and letting MSBuild take care of renaming it while moving to the `bin` directory.
+The `NServiceBus.Host.exe` creates a separate *service* [Application Domain](https://docs.microsoft.com/en-us/dotnet/framework/app-domains/application-domains) to run NServiceBus and the user code. The new domain is assigned a configuration file named after the dll that contains the class implementing `IConfigureThisEndpoint`. All the configuration should be done in that file (as opposed to `NServiceBus.Host.exe.config`). In most cases that means just adding the `app.config` file to the project and letting MSBuild rename it while moving to the `bin` directory.
```

Comment:
@mauroservienti is it worth enhancing the `IConfigureThisEndpoint` with a link to it?

Reply:
It would be nice, but that's tricky because `IConfigureThisEndpoint` doesn't exist in all versions. I'm unsure if it's possible to make the link appear only for specific versions.

Reply:
Okay so we park that thought for now.